_pkgdown.yml

```diff
@@ -0,0 +1,45 @@
+url: https://lintr.r-lib.org
+template:
+  bootstrap: 5
+
+  includes:
+    in_header: |
+      <script defer data-domain="lintr.r-lib.org,all.tidyverse.org" src="https://plausible.io/js/plausible.js"></script>
+
+reference:
+- title: Linting
+  contents:
+  - lint
+  - linters_with_defaults
+
+- title: Configuration
+  contents:
+  - read_settings
+  - exclude
+  - linters_with_tags
+  - modify_defaults
+
+- title: Individual linters
+  contents:
+  - all_undesirable_functions
+  - ends_with("linter")
+
+- title: Customer linters
+  contents:
+  - Linter
+  - expect_lint
+  - expect_lint_free
+  - ids_with_token
+  - is_lint_level
+  - use_lintr
+  - xml_nodes_to_lints
+
+- title: internal
+  contents:
+  - ends_with("linters")
```

Comment:
Now that I've played with the package a bit more I better understand the role of these linters tag pages. I think they're great as cross-reference pages, but it's not that useful to link to them from the reference index, so I've moved them to the internal section that's is not shown.